salt/haproxy/haproxy.cfg.jinja

```diff
@@ -0,0 +1,33 @@
+{%- if "kube-master" in salt['grains.get']('roles', []) -%}
+{%- set bind_ip = "0.0.0.0" -%}
+{%- else -%}
+{%- set bind_ip = "127.0.0.1" -%}
+{%- endif -%}
+global
+        log /dev/log    local0
+        log /dev/log    local1 notice
+        daemon
+
+defaults
+        log     global
+        mode    tcp
+        option  tcplog
+        option  dontlognull
+        timeout connect 5000
+        timeout client 50000
+        timeout server 50000
+
+listen kubernetes-master
+        bind {{ bind_ip }}:6443
+        mode tcp
+        default-server inter 10s fall 3
+        balance roundrobin
+
+{%- for minion_id, addrlist in salt['mine.get']('roles:kube-master', 'network.ip_addrs', 'grain').items() %}
+{% if addrlist is string -%}
+{% set addr = addrlist -%}
+{% else -%}
+{% set addr = addrlist|first -%}
+{% endif -%}
```

Comment:
Ah, a leftover from the original commit to another branch a few months back. I'll remove it.